cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/KafkaClusterState.java

```diff
@@ -204,6 +207,12 @@ protected void populateKafkaBrokerState(Map<Integer, Integer> leaderCountByBroke
         leaderCountByBrokerId.put(nodeId, 0);
       }
     }
+    // Part-3: Gather controller information.
+    replicaCountByBrokerId.keySet().forEach(brokerId -> isControllerByBrokerId.put(brokerId, false));
```

Comment:
how about brokers without replicas? Should we also include them here?

Reply:
`replicaCountByBrokerId` includes both brokers with and without replicas -- i.e. has `0` replica count. Am I missing something?

Reply:
Ah, I missed in part 2, we actually added to map.